prep-javascript-conditionals/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript Conditionals</title>
+</head>
+<body>
+  <script>
+    var age = 27;
+    if (age > 18) {
+      console.log('adult');
+    } else if (age < 18) {
+      console.log('a child');
+    }
+var student1Score = 70;
+    if (student1Score >= 70) {
+      console.log('Congrats you recieved a passing grade!');
+    } else {
+      console.log('Oh no, you failed!');
+    }
+    var student2Score = 90;
+    if (student2Score < 60) {
+      console.log('F');
+    } else if (student2Score < 70) {
+      console.log('D');
+    } else if (student2Score < 80) {
+      console.log('C');
+    } else if (student2Score < 90) {
+      console.log('B');
+    } else if (student2Score < 100) {
+      console.log('A');
+    } else {
+      console.log('A++');
+    }
+
+var season = 'autumn';
+console.log(1 === 1);{
+  if ('summer === summer'); {
+  console.log('it is hot today');
+} if ('spring === spring'); {
+  console.log('the flowers are blooming');
+} if ('autumn === autumn'); {
+  console.log('the leaves are changing colors');
+} if ('winter === winter'); {
+  console.log('it is cold today');
+}
+var dayOfTheWeek = 'Monday';
+console.log(1 === 1); {
+  if (dayOfTheWeek === 'sunday');
+  console.log('have a good weekend!');
+}
+if (dayOfTheWeek === 'Monday'); {
+console.log('Its a weekday!');
+}
```

Comment:
Here we can check if `dayOfTheWeek` is `'sunday'` or `'saturday'`. Otherwise, it's a week day.

```js
if (dayOfTheWeek === 'sunday' || dayOfTheWeek === 'saturday') {

} else {
  // it's not saturday or sunday
}
```